css/dashboards/gui.css

```diff
@@ -682,19 +682,23 @@
     display: block;
 }
 
+.highcharts-dashboards-edit-grid-items {
+    margin-top: 20px;
+}
+
 .highcharts-dashboards-edit-grid-items div {
-    width: calc(50% - 20px);
+    width: calc(100% - 20px);
```

Comment:
maybe there is a way to avoid using calc?
Why don't we go for flexbox and, something like `align-items: streach` + margins on the surrounding div?